esx_service/cli/vmdkops_admin_test.py

```diff
@@ -721,11 +733,13 @@ def test_tenant_access(self):
         # change allow_create to True
         # change max_volume size to 1000MB
         # change total_volume size to 2GB
+        volume_maxsize_in_MB = convert.convert_to_MB("1000MB")
+        volume_totalsize_in_MB = convert.convert_to_MB("3GB")
```

Comment:
Maybe we can take this as a further enhancement, but I have no plan to make change in this PR.